packages/perseus/src/styles/widgets/image.less

```diff
@@ -52,3 +52,32 @@
         margin-top: 5px;
     }
 }
+
+// Image widgets outside of hints are always centred
+.perseus-image-widget:not(.perseus-hint-renderer *) {
```

Comment:
This uses a Selectors Level 4 version of `:not()`. I had a hard time verifying if it was fully supported (caniuse.com was a bit ambiguous about browser support for the Level 4 version specifically). 

I tried on the latest of Safari, Chrome, and Firefox and it worked in all three. 🤷‍♂️